### PR TITLE
Utilise Closed Transitions Defined in Config

### PR DESCRIPTION
**Pull request by Jake151**
In the event that a jira project is using custom closed status this fix will pull those status into the Jira queries to reduce Jira overhead in pulling in issues that aren't required.

This is the same as #172, I just moved the commit to a branch on my fork as I had more to PR.

### CODE REVIEW

**Review by alpesh-te — APPROVED** [review]
It is trivial fixed. Looks good to me

**Comment by Jake151** [comment]
Hi @alpesh-te - I don't have write access to the repo so can't merge this

**Comment by kmplngj** [comment]
@SteveMcGrath Could you merge this?

**Review by SteveMcGrath — APPROVED** [review]

**Pull request merged**
